CHANGELOG.md

```diff
@@ -123,5 +124,23 @@ property should now be used instead for the `EscapeOutput` sniff.
 
 ## [0.3.0] - 2014-12-11
 
+See the comparison for full list.
+
 ### Changed
 - Use semantic version tags for releases.
+
+## [2013-10-06]
+
+See the comparison for full list.
+
+## 2013-06-11
+
+Initial tagged release.
+
+[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.7.0...HEAD
```

Comment:
Not all unreleased changes are in `develop` though. The ones I specifically mention in that list are unmerged pull requests.